components/edit-collective/sections/InvoicesReceipts.js

```diff
@@ -54,63 +95,189 @@ const InvoicesReceipts = ({ collective }) => {
+        {!showAlternativeReceiptsSection && (
+          <StyledButton
+            buttonStyle="secondary"
+            mt="24px"
+            mb="24px"
+            maxWidth={209}
+            pt="7px"
+            pb="7px"
+            pl="18px"
+            pr="16px"
+            onClick={() => setShowAlternativeReceiptsSection(true)}
+          >
+            <Plus size={14} color="#1869F5" />
+            <FormattedMessage defaultMessage="Add alternative receipt" />
+          </StyledButton>
+        )}
+        {showAlternativeReceiptsSection && (
+          <Container mt="26px" mb="24px">
+            <Flex flexWrap="wrap" flexDirection="column" width="100%">
+              <Box color="black.800" fontSize="16px" fontWeight={700} lineHeight="24px">
+                <FormattedMessage defaultMessage="Receipt title" />
+              </Box>
+              <StyledInput
+                placeholder="Custom Receipt"
+                defaultValue={alternativeReceiptTitle}
+                onChange={e => onChange(e.target.value, setAlternativeReceiptTitle)}
+                width="100%"
+                maxWidth={414}
+                mt="6px"
+              />
+              <Flex justifyContent="space-between" flexDirection={['column', 'row']} pt="26px">
+                <Box color="black.800" fontSize="16px" fontWeight={700} lineHeight="24px">
+                  <FormattedMessage defaultMessage="Custom Message" />
+                </Box>
+                <StyledButton
+                  buttonStyle="secondary"
+                  buttonSize="tiny"
+                  maxWidth="78px"
+                  pt="4px"
+                  pb="4px"
+                  pl="14px"
+                  pr="14px"
+                  height="24px"
+                  onClick={() => setShowPreview(true)}
+                >
+                  <Span fontSize="13px" fontWeight={500} lineHeight="16px">
+                    <FormattedMessage defaultMessage="Preview" />
+                  </Span>
+                </StyledButton>
+              </Flex>
+              <StyledTextarea
+                placeholder={intl.formatMessage(messages.extraInfoPlaceholder)}
+                defaultValue={alternativeInfo}
+                onChange={e => onChange(e.target.value, setAlternativeInfo)}
+                width="100%"
+                height="150px"
+                fontSize="13px"
+                mt="14px"
+              />
+            </Flex>
+            <StyledButton
+              buttonStyle="danger"
+              style={{ backgroundColor: 'white', background: 'none', borderColor: '#CC2955' }}
+              mt="24px"
+              maxWidth={225}
+              pt="7px"
+              pb="7px"
+              pl="18px"
+              pr="16px"
+              onClick={() => deleteAlternativeReceipt()}
+            >
+              <Trash size={14} color="#CC2955" />
+              <Span fontSize="14px" fontWeight={500} lineHeight="18px" style={{ color: '#CC2955' }}>
```

Comment:
The alignment looks a bit off on Chrome. Also, it's better to use `css` rather than `style` to keep relying on styled-components.

![image](https://user-images.githubusercontent.com/1556356/181508357-067de30f-fb73-4562-a664-860e071c3a47.png)

Reply:
Done at, https://github.com/opencollective/opencollective-frontend/pull/8057